lib/src/clojure_lsp/queries.clj

```diff
@@ -363,13 +363,12 @@
              rf-some
              (get-in db [:analysis filename :locals])))
 
-(defmethod find-definition :keywords
+(defmethod find-definition :keyword-usages
   [db element]
   (or (when (:ns element)
```

Comment:
It's possible to register keywords without a namespace, but this `when` means we wouldn't find those registrations. I think this is a good time to fix that. WDYT @ericdallo?

Reply:
Yes, To be hones I don't recall why we did that 🤔

Reply:
Confirmed, this is pretty easy to fix... I'll submit a follow-up PR after this one is merged.